packages/headless/package.json

```diff
@@ -30,14 +30,15 @@
     "doc:parse": "ts-node --project ./doc-parser/tsconfig.build.json ./doc-parser/doc-parser.ts"
   },
   "dependencies": {
-    "@coveo/bueno": "^0.14.0",
+    "@coveo/bueno": "^0.13.0",
```

Comment:
Bueno as no 0.14.0 version, the latest is 0.14.0-alpha.3, I'm not sure why it was there when it can't be installed
https://www.npmjs.com/package/@coveo/bueno

Reply:
Also the last officially released Bueno is 0.1.0

Reply:
Deleted every modules folder and reinstalled, it worked, but it still seems weird 🤔